codespell_lib/data/dictionary.txt

```diff
@@ -23257,6 +23272,10 @@ materils->materials
 materla->material
 materlas->materials
 mathamatics->mathematics
+mathc->match
```

Comment:
Could it be `math` or `maths` too?

If all occurrences in the wild are actually `match` typos, keep this as is.